packages/nc-gui/composables/useSharedView.ts

```diff
@@ -103,6 +116,7 @@ export function useSharedView() {
     const { data } = await $api.public.dataList(
       sharedView.value.uuid!,
       {
+        limit,
```

Comment:
here we may check the view type. If it is map view, then we conditionally pass the limit.